docs/user-guide/configure-zowe-runtime.md

```diff
@@ -381,94 +381,108 @@ TSO user IDs using Zowe must have permission to access the z/OSMF services that
 
 ## The Zowe Cross Memory Server
 
-The Zowe Cross Memory Service is a started task angel that runs an authorized server application providing privileged cross-memory services to Zowe.  It must be installed, configured and launched for the Zowe desktop to be able to operate.  It is not required for the Zowe API Mediation Layer to be launched and able to operate.   
+The Zowe Cross Memory Server is a started task angel that runs an authorized server application and its auxiliary address spaces, providing privileged cross-memory services to Zowe. To operate, the Zowe Desktop requires that the service be installed, configured, and started. The Zowe API Mediation Layer does not require the service.
 
-The server runs as a started task and requires an APF authorized load library, a program properties table (PPT) entry, and a parmlib. You can create these by using one of the following methods. The two methods achieve the same end result.
+The server and address spaces run as started tasks. To configure the service, you must create or edit APF authorized load libraries, program properties table (PPT) entries, and a parmlib. You can configure the service one two ways:
```

Comment:
`You can configure the service one two ways: ` should be `You can configure the service in one of the following two ways:`

Reply:
Done